js/src/dapps/static/console.js

```diff
@@ -18,6 +18,7 @@
 // TODO: Fix linting issues
 
 if (typeof(window.parity) == 'object')
+  window.api = window.parent.secureApi;
```

Comment:
We now do have 2 separate instances in here, one via HTTP & one via WS. 

Would make sense to have a follow-on where we converge on the secure only. i.e. The next line and friends should be changed to `api.subscribe` and the parity.js import can be removed.